docs/badges/configuration.rst

```diff
@@ -0,0 +1,241 @@
+Configuration
+=============
+
+Badges feature configuration includes:
+
+- feature configuration
+- badges management
+- requirements setup
+- backends configuration (optional)
+
+
+Feature configuration
+---------------------
+
+Badges are under a feature flag (disabled by default):
+
+.. code-block:: sh
+
+    # credentials settings:
+    ENABLE_BADGES = false
+
+Also the feature has its configuration which may look like this:
+
+.. code-block:: sh
+
+    # credentials settings:
+    BADGES_CONFIG = {
```

Comment:
To discuss: do we have any cases for configuring this in runtime (e.g., through admin panel) with fallback to the configuration from settings?